apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -437,6 +438,11 @@ mine_a_micro_block(Node) ->
 mine_a_micro_block_on_node(NName) ->
     aecore_suite_utils:mine_blocks(NName, 1, ?MINE_RATE, micro, #{}).
 
+mine_micro_blocks_on_dev1_until_empty(_Config) ->
```

Comment:
Maybe generalize with the function below? But the naming in this test module is very explicit...

Reply:
Yes, I tried to stay consistent with the existing convention

Reply:
I think the idea is that since the test flows are actually implemented at the group level, specifically named test cases make those flows more readable.